src/electionguard/wasm/ballot.cpp

```diff
@@ -0,0 +1,30 @@
+#include "electionguard/ballot.hpp"
+
+#include "../log.hpp"
+
+#include <emscripten/bind.h>
+#include <iostream>
+
+using namespace emscripten;
+using namespace electionguard;
+using namespace std;
+
+EMSCRIPTEN_BINDINGS(electionguard)
+{
+    class_<PlaintextBallot>("PlaintextBallot")
```

Comment:
this is the magic that allows the js interface to define the same names as the c++ interface